server/src/test/scala/actors/queues/DateRangeSpec.scala

```diff
@@ -52,4 +52,18 @@ class DateRangeSpec extends Specification {
       }
     }
   }
+
+  "Given a date range that spans two dates but less than 24 hours" >> {
```

Comment:
might be worth adding a test that uses dates falling in BST, and/or crossing the clock change